### PR TITLE
fix(classic): update docs for security deprecation 

### PR DESCRIPTION
**Pull request by ntindle**
We need to clearly state that the classic AutoGPT is unsupported for security reasons.

### Changes 🏗️

Adds a readme for classic autogpt to show that we won't update

#### For code changes:
- [x] I have clearly listed my changes in the PR description
- [x] I have made a test plan
- [x] I have tested my changes according to the test plan:
  
  - [ ] Test Plan

<details>
  <summary>Test Plan</summary>
  
  - [ ] Open the classic readme and make sure it renders
  - [ ] Test the links
  - [ ] Check the docs page for classic has the warning
</details>

### CODE REVIEW

**Comment by ntindle** [comment]
https://github.com/nodejs/node/issues/55829 related to failure

File: docs/content/classic/index.md [hunk 1]
```diff
@@ -8,6 +8,11 @@
 
 **Location:** `classic/original_autogpt/` in the GitHub repo
 
+**Maintance Notice:** Classic AutogGPT is not supported from a security perspective.
```
Comment (kbarnard10) [inline docs/content/classic/index.md]: AutoGPT Classic

File: docs/content/classic/index.md [hunk 2]
```diff
@@ -8,6 +8,11 @@
 
 **Location:** `classic/original_autogpt/` in the GitHub repo
 
+**Maintance Notice:** Classic AutogGPT is not supported from a security perspective. 
+Dependencies will not be updated, nor will issues be fixed. If someone wishes to
+contribute to novel development, we will give best effort merging to the changes that
+pass the existing CI.
+
 AutoGPT was conceived when OpenAI published their GPT-4 model accompanied by a paper
```
Comment (kbarnard10) [inline docs/content/classic/index.md]: AutoGPT Classic

File: classic/README.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
+# AutoGPT
+
+AutoGPT is an experimental project that demonstrates autonomous GPT-4 operation. It was designed to make GPT-4 independently operate and chain together tasks to achieve more complex goals.
```
Comment (kbarnard10) [inline classic/README.md]: AutoGPT Classic was

File: classic/README.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
+# AutoGPT
```
Comment (kbarnard10) [inline classic/README.md]: AutoGPT Classic

File: classic/README.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+## Project Status
+
+⚠️ **This project is unsupported, and dependencies will not be updated. It was an experiment that has concluded its initial research phase. If you want to use AutoGPT, you should use the [AutoGPT Platform](/autogpt_platform)**
+
+For those interested in autonomous AI agents, we recommend exploring more actively maintained alternatives or referring to this codebase for educational purposes only.
+
+## Overview
+
+AutoGPT was one of the first implementations of autonomous AI agents - AI systems that can independently:
```
Comment (kbarnard10) [inline classic/README.md]: AutoGPT Classic

File: classic/README.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+This project segment is licensed under the MIT License - see the [LICENSE](LICENSE) file for details.
+
+## Documentation
+
+Please refer to the [documentation](https://docs.agpt.co) for more detailed information about the project's architecture and concepts.
+You can browse at the same point in time as this commit so the docs don't change.
+
+## Historical Impact
+
+AutoGPT played a significant role in advancing the field of autonomous AI agents:
```
Comment (kbarnard10) [inline classic/README.md]: AutoGPT Classic

**Review by kbarnard10 — COMMENTED** [review]
Suggesting to replace mentions of "AutoGPT" with "AutoGPT Classic" when referring to the original implementation to avoid confusion with the current state of the project (platform and self-hosted), which is actively supported.

**Review by itsababseh — APPROVED** [review]
This looks good to me and is more clear with AutoGPT Classic!

**Pull request merged**